app/Http/Requests/StoreOrganisation.php

```diff
@@ -33,7 +33,6 @@ public function rules()
             'entry' => 'nullable|string',
             'type' => 'nullable|string|max:191',
             'image' => 'mimes:jpeg,png,jpg,gif,webp|max:' . Limit::upload(),
-            'location_id' => 'nullable|integer|exists:locations,id',
```

Comment:
Now we just need locations as an array to validate those, because right now I can send a locations with id 999 and it crashes the request :)